### PR TITLE
Improve packaging of web files

### PR DESCRIPTION
**Pull request by netsirius**
Make web packaging generic

### CODE REVIEW

**Review by iduartgomez — COMMENTED** [review]
remove the freenet.._old.wasm file

File: crates/locutus-dev/src/bin/local_node.rs [hunk 1]
```diff
@@ -1,7 +1,14 @@
 use clap::Parser;
```
Comment (iduartgomez) [inline crates/locutus-dev/src/bin/local_node.rs:1]: we can rename the utility to just `locutus-dev` for now

File: crates/locutus-dev/src/bin/local_node.rs [hunk 2]
```diff
@@ -30,3 +47,71 @@ async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>
...
+    Ok(())
+}
+
+fn build_view_state(
+    complete_state: &mut Vec<u8>,
+    source_path: PathBuf,
+    dest_file: PathBuf,
+) -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    tracing::debug!("Bundling `view` state from {source_path:?} into {dest_file:?}");
+    // FIXME: use instead WebModelState
```
Comment (iduartgomez) [inline crates/locutus-dev/src/bin/local_node.rs:75]: fix this

File: crates/locutus-dev/src/bin/local_node.rs [hunk 3]
```diff
@@ -30,3 +47,71 @@ async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>
...
+    state.write_all(complete_state)?;
+    Ok(())
+}
+
+fn build_model_state(
+    source_path: PathBuf,
+    dest_file: PathBuf,
+) -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    tracing::debug!("Bundling `model` contract state from {source_path:?} into {dest_file:?}");
+    // FIXME: optionally provide a path to the metadata
```
Comment (iduartgomez) [inline crates/locutus-dev/src/bin/local_node.rs:88]: fix this

Comment (netsirius) [reply crates/locutus-dev/src/bin/local_node.rs]: fixed

File: crates/locutus-dev/src/bin/local_node.rs (hunk 2 above)
Comment (netsirius) [reply crates/locutus-dev/src/bin/local_node.rs]: fixed

File: crates/locutus-dev/src/bin/local_node.rs (hunk 1 above)
Comment (netsirius) [reply crates/locutus-dev/src/bin/local_node.rs]: renamed

**Review by iduartgomez — APPROVED** [review]

**Pull request merged**